net/pdnsd/files/pdnsd.init

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh /etc/rc.common
+
+START=65
+NAME=pdnsd
+DESC="proxy DNS server"
+
+DAEMON=/usr/sbin/pdnsd
+PID_FILE=/var/run/$NAME.pid
+CACHEDIR=/tmp/pdnsd
+CACHE=$CACHEDIR/pdnsd.cache
+
+USER=nobody
+GROUP=nogroup
+
+start() {
+	echo -n "Starting $DESC: $NAME"
+
+	gen_cache
+
+	$DAEMON --daemon -p $PID_FILE
+	echo " ."
+}
+
+stop() {
+	echo -n "Stopping $DESC: $NAME"
+	kill `cat $PID_FILE` > /dev/null 2>&1
```

Comment:
Replace `` with $(). Would be a good idea to run shellcheck too.

Reply:
could this  just not just use a procd script instead?  it just starts a single process, pidfile is only used for killing.

Reply:
@karlp I don't understand. Can you provide more details and examples, thank you.